Tests/test_image_rotate.py

```diff
@@ -96,6 +96,11 @@ def test_center(self):
         self.rotate(im, im.mode, 45, center=(0, 0))
         self.rotate(im, im.mode, 45, translate=(im.size[0]/2, 0))
         self.rotate(im, im.mode, 45, center=(0, 0), translate=(im.size[0]/2, 0))
+        
+    def test_rotate_with_fill(self):
+        im = hopper()
+        target = Image.open('Tests/images/hopper45withfill.png')
+        self.assert_image_equal(im.rotate(45, fillcolor='white'), target)
```

Comment:
Hmm, tests are failing due to "got different content", I guess from  minor differences from the Hopper photo.

Is it any better with with plain colour fields, something like this?

```python
    def test_rotate_no_fill(self):
        # Arrange
        im = Image.new('RGB', (100, 100), 'green')
        target = Image.open('Tests/images/rotate_45_no_fill.png')

        # Act
        im = im.rotate(45)

        # Assert
        self.assert_image_equal(im, target)

    def test_rotate_with_fill(self):
        # Arrange
        im = Image.new('RGB', (100, 100), 'green')
        target = Image.open('Tests/images/rotate_45_with_fill.png')

        # Act
        im = im.rotate(45, fillcolor='white')

        # Assert
        self.assert_image_equal(im, target)
```

Reply:
I had not added anything for no fillcolor because I guessed that scenario is already handled in the other tests.
But I will do so if required.